packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -155,6 +178,18 @@ class PermissionCollection extends DocumentCollection {
   }
 }
 
+/**
+ * Is the document a collection document?
+ * @param {{_type}} document
+ * @returns {boolean}
+ */
+export const isCollection = document => {
+  if (isFile(document)) return false
+  if (document._type == 'io.cozy.notes') return false
+  if (document._type == 'io.cozy.notes.events') return false
+  return true
+}
```

Comment:
I don't understand what is a collection in this context ? How should one use this method ?